templates/restaurant/list.html.twig

```diff
@@ -32,4 +122,26 @@
 {% block scripts %}
   {{ encore_entry_script_tags('search-address') }}
   {{ encore_entry_script_tags('restaurant-list') }}
+
+  <script>
```

Comment:
Instead of changing `window.location.href`, we could wrap the filters in a `<form method="get">`. 
Then the JavaScript code would just have to submit the form when an input is toggled.